plugins/command-line/prism-command-line.js

```diff
@@ -63,7 +63,10 @@ Prism.hooks.add('complete', function (env) {
 
 		if (!isNaN(outputStart) && !isNaN(outputEnd)) {
 			for (var j = outputStart; j <= outputEnd && j <= prompt.children.length; j++) {
-				prompt.children[j - 1].className = 'output';
+				var node = prompt.children[j - 1];
+				node.removeAttribute('data-user');
+				node.removeAttribute('data-host');
+				node.removeAttribute('data-prompt');
```

Comment:
It’s not about doing it in one line, it's about removing repetition. Actually, I wouldn't do it in one line, I would format it as 3, for readability. Anyhow, no strong opinion here.